numba/cuda/decorators.py

```diff
@@ -55,6 +55,11 @@ def jit(func_or_sig=None, device=False, inline=False, link=[], debug=None,
                 enabled. When ``True``, ``-opt=3`` is passed to NVVM. When
                 ``False``, ``-opt=0`` is passed to NVVM. Defaults to ``True``.
     :type opt: bool
+
+    In addition to the above parameters the keyword ``lineinfo=True`` may
```

Comment:
Could you follow the format of the other documented kwargs for this (i.e. `:param lineinfo:` and `:type lineinfo:`) please? I think this will make it clearer in the documentation, and this format is not checked against the list of kwargs in the function signature

Reply:
Certainly can. I wasn't sure if I should since it's not explicitly in the interface.